entries/n/n26.com.json

```diff
@@ -5,12 +5,22 @@
       "sms",
       "custom-software"
     ],
+    "custom-software": [
+      "N26 app"
+    ],
     "documentation": "https://support.n26.com/en-eu/security/passwords-and-codes/how-to-log-in-with-two-factor-authentication",
     "keywords": [
       "banking"
     ],
     "regions": [
-      "de"
+      "at",
+      "br",
+      "fr",
+      "de",
+      "it",
+      "es",
+      "ch",
+      "us"
```

Comment:
Also: https://n26.com/en-de/press/press-release/n26-to-discontinue-us-operations

```suggestion
```